docs/reference/commandline/build.md

```diff
@@ -110,12 +110,6 @@ Build Syntax Suffix             | Commit Used           | Build Context Used
 `myrepo.git#mytag:myfolder`     | `refs/tags/mytag`     | `/myfolder`
 `myrepo.git#mybranch:myfolder`  | `refs/heads/mybranch` | `/myfolder`
 
-> **Note**
->
-> You cannot specify the build-context directory (`myfolder` in the examples above)
-> when using BuildKit as builder (`DOCKER_BUILDKIT=1`). Support for this feature
```

Comment:
If this is specifically for `DOCKER_BUILDKIT=1` then it has not been updated in dockerd release yet. We could add that it is supported since v0.9, currently supported via buildx.